lib/pcm_stream_channel_modifier/Kconfig

```
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig PSCM
+	bool "PSCM - PCM stream channel modifier library"
```

```suggestion
	bool "PSCM - PCM Stream Channel Modifier library"
```

The .rst file uses capitalized letters

---

Same on line 15